src/EditorFeatures/CSharpTest/DocumentationComments/DocumentationCommentTests.cs

```diff
@@ -706,6 +706,26 @@ public void TypingCharacter_NotInsideCtor()
             VerifyTypingCharacter(code, expected);
         }
 
+        [WpfFact, Trait(Traits.Feature, Traits.Features.DocumentationComments)]
+        public void TypingCharacter_NotInTopLevel()
+        {
+            var code = @"
+using System;
+
+//$$
+Console.WriteLine();
+";
+
+            var expected = @"
+using System;
+
+///$$
+Console.WriteLine();
+";
+
+            VerifyTypingCharacter(code, expected);
+        }
```

Comment:
consider adding a test for namespaces too.

Reply:
➡️ This test is now added